crates/mysticeti-core/src/core.rs

```diff
@@ -320,8 +325,21 @@ impl<H: BlockHandler> Core<H> {
         }
 
         tracing::debug!("Created block {block:?}");
+        if self.is_attacked(){
+            // sleep for 1 second
+            std::thread::sleep(std::time::Duration::from_millis(Self::ATTACK_DELAY));
```

Comment:
This will stall the main tread rather than only the messages received by other peers. This means that it will also also slow down blocks generation; and various tasks may call `try_new_block` many times (potentially every time they receive or create a block (this sleep may thus be triggered multiple times per round)